node/src/chain_spec.rs

```diff
@@ -111,14 +112,11 @@ pub fn local_testnet_config(id: ParaId) -> ChainSpec {
 
 	ChainSpec::from_genesis(
 		// Name
-		"Canvas Local",
+		"Canvas",
 		// ID
-		"canvas_local",
+		"canvas_rococo",
```

Comment:
```suggestion
		"canvas-rococo",
```

Consistency with the protocol ID.